app/views/pages/_settings_form.slim

```diff
@@ -27,17 +27,19 @@
         = label_with_tooltip(f, :notes, t('pages.edit.notes'), t('tooltips.notes'))
         = f.text_area :notes, class: 'form-control'
 
-    - if page.ak_petition_resource_uri.present? || page.ak_donation_resource_uri.present?
+    - if Settings.ak_report_url.present?
```

Comment:
How come we went from checking the resource URIs on the page to checking this in Settings? How do you get the values for `pages.edit.ak_donation_report` and `pages.edit.ak_petition_report`?

Reply:
* The check for the resource URI's is done below this. Here I'm checking for the ak_report_url to keep it optional in case AK is not being used, or even if it's used, there's no Report to link to.
* The translation keys are in the translation file.

Reply:
Ok, thanks! :)